Open Judge System/Web/OJS.Web/Areas/Api/Controllers/ResultsController.cs

```diff
@@ -4,11 +4,12 @@
     using System.Globalization;
     using System.Linq;
     using System.Web.Mvc;
-
+    using MissingFeatures;
```

Comment:
Do not use this. We  must try to remove it  completely

Reply:
Its an unused import so i'll just delete it